src/client/components/general/ProductCard/index.js

```diff
@@ -2,16 +2,22 @@ import React from "react";
 import { Button, Tag } from "../../index";
 import "./productCard.scss";
 
-function ProductCard({ title, description, price, img, alt, tag }) {
+function ProductCard({ id, title, description, price, image, category, add}) {
   
   return (
     <article className="product">
-      {tag && <Tag text={tag.text} theme={tag.theme} />}
-      <img src={img} alt={alt} className="product-img"></img>
+      {category === "jewelery" ? (
+        <Tag text="New!" theme="blue" />
+      ) : title.includes("Women") ? (
+        <Tag text="15% off" theme="beige" />
+      ) : "" }
```

Comment:
Another way of doing this logic (from line 9 to 13) would be like this:
`{ (category === 'jewelery' || title.includes('Women')) && 
    <Tag text={ category === 'jewelery' ? 'New!' : '15% off' } theme={ category === 'jewelery' ? 'blue' : 'beige' } /> 
}`
This way you wouldn't have to leave those empty strings at the end of the ternary operator, returning nothing